contrib/platform/test/com/sun/jna/platform/win32/COM/ShellApplicationWindowsTest.java

```diff
@@ -70,9 +100,13 @@ public void testWindowsCount()
         {
             // For reasons unknown, Shell.Application.Windows can have null members inside it.
             // All I care about is whether or not the collection contains the window I opened.
-            if (ie != null && "about:blank".equals(ie.getURL()))
+            if (ie != null)
             {
-                pageFound = true;
+                String url = ie.getURL();
+                System.out.println(url);
```

Comment:
This (the whole diff around it actually) seems left over from debugging?

Reply:
Yes - will be reverted.